db/seeds.rb

```diff
@@ -14,3 +14,17 @@
 category2.videos << Video.new(title: "Air Force 1", description: "Stunning", small_cover_url: "/tmp/family_guy.jpg", large_cover_url: "/tmp/monk_large.jpg")
 category1.videos << Video.new(title: "Notebook", description: "I cried", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
 category1.videos << Video.new(title: "Yes Man", description: "Must see", small_cover_url: "/tmp/south_park.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Deep Impact", description: "Wow, a gret movie", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Air Force 1", description: "Stunning", small_cover_url: "/tmp/family_guy.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Notebook", description: "I cried", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Yes Man", description: "Must see", small_cover_url: "/tmp/south_park.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Deep Impact", description: "Wow, a gret movie", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Air Force 1", description: "Stunning", small_cover_url: "/tmp/family_guy.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Notebook", description: "I cried", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Yes Man", description: "Must see", small_cover_url: "/tmp/south_park.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Deep Impact", description: "Wow, a gret movie", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category2.videos << Video.new(title: "Air Force 1", description: "Stunning", small_cover_url: "/tmp/family_guy.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Notebook", description: "I cried", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg")
+category1.videos << Video.new(title: "Yes Man", description: "Must see", small_cover_url: "/tmp/south_park.jpg", large_cover_url: "/tmp/monk_large.jpg")
```

Comment:
There is a better way than this, you can pass category to video when you create a Video object.

```ruby
Video.create(title: .., ..., category: category1)
```

Its much easier to read too and you can use associations to empower your code.